src/main.py

```
@@ -0,0 +1,32 @@
+
+    # initialize Preprocessor
+    data_processor = Preprocessor(filename='hotel_reservations.csv', config=config)
+    logger.info("DataProcessor initialized.")
+
+    # execute sci-kit learn pipeline to perform preprocessing tasks
+    data_processor.preprocess_raw_data()
+    logger.info("Data preprocessed.")
+
+    # Split the data
+    X_train, X_test, y_train, y_test = data_processor.split_data()
+    logger.info(f"Data split into training and test sets.")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)